### PR TITLE
tiflash: add descriptions for some conf items under [flash] section

### PR DESCRIPTION
**Pull request by CalvinNeo**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v7.4 (TiDB 7.4 versions)
- [ ] v7.3 (TiDB 7.3 versions)
- [ ] v7.2 (TiDB 7.2 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v7.0 (TiDB 7.0 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s): https://github.com/pingcap/tiflash/issues/7564

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tiflash/tiflash-configuration.md [hunk 1]
```diff
@@ -131,6 +131,15 @@ delta_index_cache_size = 0
...
     service_addr =  TiFlash raft 服务 和 coprocessor 服务监听地址
 
+    ## 从 v7.4.0 引入，当当前 Raft 状态机推进的 applied_index 和上次落盘时的 applied_index 相差高于 compact_log_min_gap 时，
+    ## 不再拒绝来自 TiKV 的 CompactLog 命令，并进行落盘。调大该值会减少落盘频率，增大内存开销。调小该值不会让 TiFlash 落盘频率高于 TiKV。
+    ## 建议保持默认值。
+    compact_log_min_gap = 默认值 200
+    ## 从 v5.0 就存在，当 TiFlash 缓存的 Region 行数或者大小超过以下任一阈值时，不再拒绝来自 TiKV 的 CompactLog 命令，并进行落盘。
+    ## 建议保持默认值。
+    compact_log_min_rows = 默认值 40k
+    compact_log_min_bytes = 默认值 32MB
```
Comment (JaySon-Huang) [inline tiflash/tiflash-configuration.md:141]:
```suggestion
    ## 不再拒绝来自 TiKV 的 CompactLog 命令，并进行落盘。调大该值会减少落盘频率，增大内存开销。调小该值不会让 TiFlash 落盘频率高于 TiKV。
    ## 建议保持默认值 200。
    # compact_log_min_gap = 200
    ## 从 v5.0 就存在，当 TiFlash 缓存的 Region 行数或者大小超过以下任一阈值时，不再拒绝来自 TiKV 的 CompactLog 命令，并进行落盘。
    ## 建议保持默认值。
    # compact_log_min_rows = 40960
    # compact_log_min_bytes = 33554432
```

**Review by JaySon-Huang — APPROVED** [review]
LGTM

File: tiflash/tiflash-configuration.md [hunk 2]
```diff
@@ -131,6 +131,15 @@ delta_index_cache_size = 0
     tidb_status_addr = tidb status 端口地址 # 多个地址以逗号分割
     service_addr =  TiFlash raft 服务 和 coprocessor 服务监听地址
 
+    ## 从 v7.4.0 引入，当当前 Raft 状态机推进的 applied_index 和上次落盘时的 applied_index 相差高于 compact_log_min_gap 时，
+    ## 不再拒绝来自 TiKV 的 CompactLog 命令，并进行落盘。调大该值会减少落盘频率，增大内存开销。调小该值不会让 TiFlash 落盘频率高于 TiKV。
+    ## 建议保持默认值。
+    # compact_log_min_gap = 200
+    ## 从 v5.0 就存在，当 TiFlash 缓存的 Region 行数或者大小超过以下任一阈值时，不再拒绝来自 TiKV 的 CompactLog 命令，并进行落盘。
```
Comment (JaySon-Huang) [inline tiflash/tiflash-configuration.md:138]:
```suggestion
    ## 从 v5.0 引入，当 TiFlash 缓存的 Region 行数或者大小超过以下任一阈值时，不再拒绝来自 TiKV 的 CompactLog 命令，并进行落盘。
```

当作是 v5.0 引入好了

File: tiflash/tiflash-configuration.md [hunk 3]
```diff
@@ -131,6 +131,15 @@ delta_index_cache_size = 0
     tidb_status_addr = tidb status 端口地址 # 多个地址以逗号分割
     service_addr =  TiFlash raft 服务 和 coprocessor 服务监听地址
 
+    ## 从 v7.4.0 引入，当当前 Raft 状态机推进的 applied_index 和上次落盘时的 applied_index 相差高于 compact_log_min_gap 时，
```
Comment (qiancai) [inline tiflash/tiflash-configuration.md:134]:
```suggestion
    ## 从 v7.4.0 引入，在当前 Raft 状态机推进的 applied_index 和上次落盘时的 applied_index 的差值高于 compact_log_min_gap 时，
```

File: tiflash/tiflash-configuration.md [hunk 4]
```diff
@@ -131,6 +131,15 @@ delta_index_cache_size = 0
...
+    ## 不再拒绝来自 TiKV 的 CompactLog 命令，并进行落盘。调大该值会减少落盘频率，增大内存开销。调小该值不会让 TiFlash 落盘频率高于 TiKV。
```
Comment (qiancai) [inline tiflash/tiflash-configuration.md:135]:
```suggestion
    ## 执行来自 TiKV 的 CompactLog 命令，并进行落盘。调大该值会减少落盘频率，增大内存开销；调小该值会使 TiFlash 落盘频率低于 TiKV。
```

File: tiflash/tiflash-configuration.md (hunk 2 above)
Comment (qiancai) [reply tiflash/tiflash-configuration.md:138]: 可以说下这个配置项的单位吗

File: tiflash/tiflash-configuration.md (hunk 4 above)
Comment (CalvinNeo) [reply tiflash/tiflash-configuration.md:135]: 这句话的意思是该值越小，落盘频率**可能**越低，但不会低于 tikv 的落盘频率，改成“调小该值会使 TiFlash 落盘频率低于 TiKV”的话实际就反过来了。

Comment (qiancai) [reply tiflash/tiflash-configuration.md:135]: 那“调大该值会减少落盘频率“吗？

Comment (CalvinNeo) [reply tiflash/tiflash-configuration.md:135]: 嗯，我上一个评论写错了。实际情况是：
调小该值可能会让落盘频率变高，也可能不会。但无论如何，TiFlash 落盘频率不会高于 TiKV。

**Review by qiancai — APPROVED** [review]

**Review by joey-yez — APPROVED** [review]
# compact_log_min_rows = 40960 # 40k
    # compact_log_min_bytes = 33554432 # 32MB
These two configurations already exist. Only `compact_log_min_gap` is new added.

File: tiflash/tiflash-configuration.md [hunk 5]
```diff
@@ -131,6 +131,15 @@ delta_index_cache_size = 0
     tidb_status_addr = tidb status 端口地址 # 多个地址以逗号分割
     service_addr =  TiFlash raft 服务 和 coprocessor 服务监听地址
 
+    ## 从 v7.4.0 引入，在当前 Raft 状态机推进的 applied_index 和上次落盘时的 applied_index 的差值高于 compact_log_min_gap 时，
+    ## TiFlash 将执行来自 TiKV 的 CompactLog 命令，并进行数据落盘。调大该差值可能降低 TiFlash 的落盘频率，从而减少随机写场景下的读延迟，但会增大内存开销。调小该差值可能提升 TiFlash 的落盘频率，从而缓解 TiFlash 内存压力。但无论如何，在目前阶段，TiFlash 的落盘频率不会高于 TiKV，即使设置该差值为 0。
+    ## 建议保持默认值。
+    # compact_log_min_gap = 200
```
Comment (hfxsd) [inline tiflash/tiflash-configuration.md:137]: 默认值和取值范围是多少

File: tiflash/tiflash-configuration.md [hunk 6]
```diff
@@ -131,6 +131,15 @@ delta_index_cache_size = 0
...
+    ## 从 v5.0 引入，当 TiFlash 缓存的 Region 行数或者大小超过以下任一阈值时，TiFlash 将执行来自 TiKV 的 CompactLog 命令，并进行落盘。
+    ## 建议保持默认值。
```
Comment (hfxsd) [inline tiflash/tiflash-configuration.md:139]: 加上默认值和取值范围

**Review by hfxsd — APPROVED** [review]

**Review by qiancai — APPROVED** [review]

**Pull request merged**